src/app/menubar/components/menubar-container/menubar-container.component.ts

```diff
@@ -23,7 +23,11 @@ export class MenubarContainerComponent implements OnInit {
 
   public logout() {
     this.authService.logout();
-    this.router.navigateByUrl('');
+    this.router.navigateByUrl('/login');
+  }
+
+  public login() {
+    this.router.navigateByUrl('/login');
   }
```

Comment:
waarom gebruik niet gewoon routerlink zoals het eerst stond? dan is die methode niet eens nodig